<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 
   Palette:
   - bg-dark      #171a1d
   - panel        #20252a
   - panel-2      #242a30
   - text         #e8e6e3
   - text-dim     #b8b4ae
   - gold         #caa55b
   - gold-dark    #9b7d3f
   - emerald      #2e7d6b
   - red          #c45b5b
   - focus        #6fbfa7
 */

QMainWindow, QWidget#centralwidget {
    background: #171a1d;
    color: #e8e6e3;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Roboto&quot;, Arial;
    font-size: 14px;
}

/* Кнопки по умолчанию */
QPushButton {
    background: #242a30;
    color: #e8e6e3;
    border: 1px solid #323a42;
    border-radius: 12px;
    padding: 8px 16px;
    min-height: 34px;
}
QPushButton:hover {
    background: #2b3238;
    border-color: #3a444e;
}
QPushButton:pressed {
    background: #23292f;
    border-color: #46525e;
}
QPushButton:disabled {
    color: #7b7e82;
    background: #1f2327;
    border-color: #2a3036;
}

QPushButton#btnConnect,
QPushButton#btnEnterData {
    background: #2e7d6b;
    border: 1px solid #2c6f60;
}
QPushButton#btnConnect:hover,
QPushButton#btnEnterData:hover {
    background: #338a76;
    border-color: #2f7b6a;
}
QPushButton#btnConnect:pressed,
QPushButton#btnEnterData:pressed {
    background: #28715f;
}

QPushButton#btnCreateSchema,
QPushButton#btnShowData {
    background: #caa55b;
    color: #171a1d;
    border: 1px solid #9b7d3f;
}
QPushButton#btnCreateSchema:hover,
QPushButton#btnShowData:hover {
    background: #d6b473;
}
QPushButton#btnCreateSchema:pressed,
QPushButton#btnShowData:pressed {
    background: #b89149;
}

QPushButton#btnOpenLog, QPushButton#btnAbout, QPushButton#btnExit {
    background: #242a30;
}
QPushButton#btnExit:hover { border-color: #c45b5b; color: #ffdddd; }
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbout">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>О программе</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenLog">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Открыть лог-файл</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnShowData">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>Показать данные</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUpdate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnAddClient">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string>Добавить клиента</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddRoom">
          <property name="text">
           <string>Добавить номер</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddStays">
          <property name="text">
           <string>Добавить размещение</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Подключится</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreateSchema">
        <property name="text">
         <string>Создать схему</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
